src/opera_disp_tms/create_tile_map.py

```diff
@@ -51,30 +54,22 @@ def create_tile_map(measurement_type: str, input_rasters: list[Path]) -> Path:
         measurement_type: Data measurement type to set scale_range and output folder
         input_rasters: List of gdal-compatible raster paths to mosaic
     """
-
+    scale_range = SCALE_DICT[measurement_type]
     output_dir = Path(measurement_type)
 
     with tempfile.NamedTemporaryFile() as mosaic_vrt, tempfile.NamedTemporaryFile() as byte_vrt:
         input_raster_strs = [str(raster) for raster in input_rasters]
 
         # mosaic the input rasters
         gdal.BuildVRT(mosaic_vrt.name, input_raster_strs, resampleAlg='nearest')
-
-        # scale the mosaic from Float to Byte
-        vrt_info = gdal.Info(mosaic_vrt.name, stats=True, format='json')
-        stats = vrt_info['bands'][0]['metadata']['']
-
-        if measurement_type == 'displacement':
-            scale_range = [float(stats['STATISTICS_MINIMUM']), float(stats['STATISTICS_MAXIMUM'])]
-        else:
-            scale_range = [-0.05, 0.05]
+        vrt_info = gdal.Info(mosaic_vrt.name, format='json')
 
         gdal.Translate(
             destName=byte_vrt.name,
             srcDS=mosaic_vrt.name,
             format='VRT',
             outputType=gdalconst.GDT_Byte,
-            scaleParams=[scale_range],
+            scaleParams=[[*scale_range, 1, 255]],
```

Comment:
Just want to confirm that the nested list structure here (e.g. `[[-0.25, 0.25, 1, 255]]`) is correct?

Reply:
Yes, see https://gdal.org/en/stable/api/python/utilities.html#osgeo.gdal.TranslateOptions